builder/metadata.go

```diff
@@ -1,44 +1,118 @@
 package builder
 
 import (
+	"fmt"
+
 	"github.com/buildpack/pack/lifecycle"
+	"github.com/buildpack/pack/style"
 )
 
 const MetadataLabel = "io.buildpacks.builder.metadata"
 
 type Metadata struct {
 	Description string              `json:"description"`
 	Buildpacks  []BuildpackMetadata `json:"buildpacks"`
-	Groups      []GroupMetadata     `json:"groups"`
+	Groups      OrderMetadata       `json:"groups"`
 	Stack       StackMetadata       `json:"stack"`
 	Lifecycle   lifecycle.Metadata  `json:"lifecycle"`
 }
 
 type BuildpackMetadata struct {
 	ID      string `json:"id"`
 	Version string `json:"version"`
-	Latest  bool   `json:"latest"`
+	Latest  bool   `json:"latest"` // deprecated
 }
 
-type GroupMetadata struct {
-	Buildpacks []GroupBuildpack `json:"buildpacks" toml:"buildpacks"`
+type OrderMetadata []GroupMetadata
+
+func (o OrderMetadata) ToConfig() OrderConfig {
```

Comment:
This appears to be translating between two identical objects

Reply:
Not necessarily, the inner object might have that resemblance but the parent `Order` differs in inner properties. `Buildpacks` vs `Groups`.